app/grandchallenge/algorithms/templates/algorithms/algorithm_detail.html

```diff
@@ -66,6 +66,13 @@
                 </a>
             {% endif %}
 
+            {% if request.user.is_staff and "execute_algorithm" in algorithm_perms and object.latest_ready_image %}
```

Comment:
Rather than limiting this to staff, should we have a boolean on the `Algorithm` that switches between the two modes? Default use the legacy, but then we can turn this feature on for specific algorithms. Only make the boolean available in the Django admin.

Reply:
I like this idea.